packages/cli/src/util/get-update-command.ts

```diff
@@ -100,13 +82,24 @@ async function isGlobal() {
 export default async function getUpdateCommand(): Promise<string> {
   const pkgAndVersion = `${getPkgName()}@latest`;
 
+  const entrypoint = await realpath(process.argv[1]);
+  let { cliType, lockfilePath } = await scanParentDirs(
+    dirname(dirname(entrypoint))
+  );
+  if (!lockfilePath) {
+    // Global installs for npm do not have a lockfile
+    cliType = 'npm';
+  }
+  const yarn = cliType === 'yarn';
+
+  let install = yarn ? 'add' : 'i';
```

Comment:
I think `pnpm add` should be used too, right?

Reply:
Both work, but ya from the docs it looks like `add` would be more appropriate for this use-case.